README.md

```diff
@@ -144,9 +151,9 @@ struct simplefs_extent
 ## TODO
 
 - Bugs
```

Comment:
Don't change the items in the pull request. Instead, you can send another to improve markdown usage.